src/buttons.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { useState } from "react";
+import swal from "sweetalert";
+
+function TaskForm() {
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+
+  const showAlert = () => {
+    swal({
+      title: "confirm",

```

Comment:
It would be better to use 'Confirm'

It looks weird for the user if this is in lowercase